source/locale/en/symbols.dic

```diff
@@ -13,9 +13,10 @@ complexSymbols:
 ; phrase ending	(?<=[^\s;]);(?=\s|$)
 : phrase ending	(?<=[^\s:]):(?=\s|$)
 # Others
-decimal point	(?<![^\d -])\.(?=\d)
+decimal point	(?<![^\d -])(?P<decimal>\.)(?=\d)
 in-word '	(?<=[^\W_])['’]
 negative number	(?<!\w)-(?=[$£€¥.]?\d)
+thousands separator	(?<=\d)\,(?=\d{3})

```

Comment:
Part of what made this challenging is that hindi I think and some languages has numbers grouped in ways that I don't understand. I.e. 885 85 84 93 47. bhavia knows more about these languages. He mentioned in 4273.